test/classes/ferguson/inherit-generic-q.compopts

```diff
@@ -0,0 +1 @@
+--warn-unstable
```

Comment:
I don't think we should only test the warning with `--warn-unstable`.  Seems like an appropriate time for separate .good files and multiple lines in the .compopts file

Reply:
Done